eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -149,6 +149,8 @@ jobs:
     CmakeArgs: ""
     AZURE_TEST_MODE: "LIVE"
     AZURE_LOG_LEVEL: "verbose"
+    # Surface the ServiceDirectory parameter as an environment variable so tests can take advantage of it.
+    AZURE_SERVICE_DIRECTORY: ${{ parameters.ServiceDirectory }}
```

Comment:
@LarryOsterman I created https://github.com/Azure/azure-sdk-tools/pull/2810 to set this in the core deployment script, which should also make it slightly easier to run locally if copy/pasting the env var outputs from that script.

Reply:
Once your PR is approved and committed, I'll remove this since it makes this change redundant.